rules-tests/CodeQuality/Rector/ClassMethod/LocallyCalledStaticMethodToNonStaticRector/Fixture/skip_commented.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\CodeQuality\Rector\ClassMethod\LocallyCalledStaticMethodToNonStaticRector\Fixture;
+
+final class SkipCommented
+{
+    public static function run()
+    {
+        $result = self::anotherStatic();
```

Comment:
the self::anotherStatic() one should be processed, as it called, that mean it is used, the verification seems on current method over the caller one.

I will check on it.

Reply:
It should not, as this method is static

Reply:
ok, it skipped due to `run()` method is `static`, so it seems nothing to do with commented code, I will check more in case of regression of the patch, since current class name call seems only on `['self', 'static']`, which current class name  for `ClassName::execute()` is not included.

Reply:
I create new PR for actual fixture for never called, with non-static method `run()`

- https://github.com/rectorphp/rector-src/pull/6732